src/test/regress/spec/isolation_move_placement_vs_move_placement.spec

```diff
@@ -46,6 +56,12 @@ step "s2-commit"
 	COMMIT;
 }
 
+step "s2-wait-and-commit"
+{
+	SELECT pg_sleep(1);
```

Comment:
Why is this needed?

Reply:
otherwise session2 immediately commits and releases the lock on the shard, so the shard move gets completed with the cleanup